pilot/cmd/pilot-agent/main.go

```diff
@@ -95,6 +95,8 @@ var (
 		"the provider of Pilot DNS certificate.").Get()
 	jwtPolicy = env.RegisterStringVar("JWT_POLICY", jwt.JWTPolicyThirdPartyJWT,
 		"The JWT validation policy.")
+	useFakeSymbolTableVar = env.RegisterBoolVar("USE_FAKE_SYMBOL_TABLE", true,
```

Comment:
I would just add PROXY_STARTUP_ARGS as a parameter.
That way we can add any param we want. We may not need this parameter for long at all.

We should be able to do  `PROXY_STARTUP_ARGS="--use-fake-symbol-table --so-other-param=5"`

You can split `PROXY_STARTUP_ARGS` on whitespace and append to `startupArgs`

Reply:
I am not sure whether that's the correct way to go with, for what you said then what's the difference between `startupArgs` and `PROXY_STARTUP_ARGS` then